src/config/config.js

```diff
@@ -228,6 +228,297 @@ export let config = {
+            'thebohojane.com',
+            'www.crazycoolcustomtees.com',
+            'hoodedwarrior.commercehq.com',
+            'hoodedwarrior.com',
+            'death-saves.com',
+            'posterprintshop.com',
+            'gabrielcosmeticsinc.com',
+            'mountainamericajerky.com',
+            'theskirtsociety.com',
+            'www.austinaquafarms.com',
+            'www.diedinhouse.com',
+            'lilipubsorders.com',
+            'rcegs.qnryj.servertrust.com',
+            'www.k12schoolsupplies.net',
+            'www.houstonhotels.org',
+            'divinebazaar.deals',
+            'www.thedivinebazaar.com',
+            'cloutcar.tel',
+            'superplastic.co',
+            'janky.backerkit.com',
+            'indianapolis.marketwagon.com',
+            'fortwayne.marketwagon.com',
+            'michiana.marketwagon.com',
+            'evansville.marketwagon.com',
+            'lovevibescrystals.com',
+            'aluminyze.pro',
+            'aluminyze.com',
+            'abdlcompany.com',
+            'ddlgboutique.com',
+            'abdltoys.com',
+            'inktale.com',
+            'nationalhighwaysafetyadministration.com',
+            'www.propellerdepot.com',
+            'admin.billoreilly.com',
+            'www.billoreilly.com',
+            '2bsilent.mykajabi.com',
+            'database.castingfrontier.com',
+            'fishandcountry.com',
+            'register.realestatewealthexpo.com',
+            'shoplabeye.com',
+            'www.lagoldleafus.com',
+            'www.oldmercs.com',
+            'www.codynolove.com',
+            'loveonestore.com',
+            'www.megtronix.com',
+            'www.computermallshop.com',
+            'www.gallerr.com',
+            'squishiecats.com',
+            'woopwoo.com',
+            'socksery.com',
+            'ossom.co',
+            'www.chibi-kingdom.com',
+            'oddlyhomey.com',
+            'es.postermywall.com',
+            'fr.postermywall.com',
+            'pt.postermywall.com',
+            'de.postermywall.com',
+            'nl.postermywall.com',
+            'www.postermywall.com',
+            'it.postermywall.com',
+            'www.edashdeals.com',
+            'edashdeals.com',
+            'edashdeals.commercehq.com',
+            'kensingtonproducts.com',
+            'spartanutrition.com',
+            'biovape.co',
+            'unicornclub.us',
+            'livebetterwith.com',
+            'homedna.com',
+            'lightsplanneraction.co',
+            'stickandpoketattookit.com',
+            'pepperhead.com',
+            'stage.crazyhotseeds.com',
+            'ca.bathmatedirect.com',
+            'fi.bathmatedirect.com',
+            'au.bathmatedirect.com',
+            'bathmatedirect.com',
+            'eu.bathmatedirect.com',
+            'nl.bathmatedirect.com',
+            'il.bathmatedirect.com',
+            'int.bathmatedirect.com',
+            'www.bathmatedirect.com',
```

Comment:
If we're stripping out the `www.` in the url in `buildFundingLogoThrottle`, we should do the same here right? Otherwise many of these urls won't match.

Reply:
Yup.  Taking out the `www.`.